test/units/plugins/filter/test_uuid.py

```diff
@@ -0,0 +1,32 @@
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import sys
+import pytest
+
+from units.compat import unittest
```

Comment:
Net new tests should be "pytest style tests" rather than "unittest style tests".

Reply:
Do you have an example I can copy the structure from? I have used netaddr

Reply:
[Here](https://github.com/ansible/ansible/blob/4048fb4da72cd0ab132f66f5ecc7d45347861f40/test/units/module_utils/common/validation/test_check_type_str.py) is an example.